src/runtime/opencl/opencl_common.h

```diff
@@ -50,14 +50,18 @@
  * files.  This also allows us to expose the OpenCL version through
  * tvm.runtime.Device.
  */
-#define CL_TARGET_OPENCL_VERSION 120
+#define CL_TARGET_OPENCL_VERSION 220
```

Comment:
Emm... Are you sure that we want to up OpenCL version for all devices? I'm afraid that we shouldn't do it.
If you wish, you can use `ifdef USE_ADRENO_RECORDING` or something like that for this purpose:
```c++
#ifdef USE_ADRENO_RECORDING
#define CL_TARGET_OPENCL_VERSION 220
#else
#define CL_TARGET_OPENCL_VERSION 120
#endif
```